### ISSUE TITLE
Prompt to select mouse or click mode does not allow 'accept' in debug mode

### ISSUE THREAD

**Issue by SteveBenz** [linuxgurugamer/ClickThroughBlocker#13]
When I'm running KSP in debug mode, when I press "Accept" it fails to do anything.  In the log file, it shows a PathNotFound exception when referencing 'GameData\000_ClickThroughBlocker\PluginData\PopUpShown.cfg' -- could it be that it's using a relative path?  Perhaps KSP gets launched with a different current directory when run from the debugger?  (could be that this can happen with different steam configurations too)

You might want to do what Kerbal Engineer (and others) do -- they use System.Reflection.Assembly.GetExecutingAssembly().Location and build a relative path off of that.

**Comment by w3bward** [comment]
I have this same issue. I do not believe that I am launching in debug mode. Clicking "Accept" does nothing, eventually I have to dismiss the window by clicking cancel instead, and the window is shown every time I launch my saved game.

Opening the console does show an exception that sounds like it's probably the same one @SteveBenz is referring to, and this appears in Player.log:

```
DirectoryNotFoundException: Could not find a part of the path "GameData/000_ClickThroughBlocker/PluginData/PopUpShown.cfg".
  at System.IO.File.Delete (System.String path) [0x00058] in <ad04dee02e7e4a85a1299c7ee81c79f6>:0 
  at ClearAllInputLocks.OneTimePopup.RemovePopUpFlagFile () [0x00000] in <4758f016d669411aa83fa2b6c8cb92c4>:0 
  at ClearAllInputLocks.OneTimePopup.CreatePopUpFlagFile () [0x00000] in <4758f016d669411aa83fa2b6c8cb92c4>:0 
  at ClearAllInputLocks.OneTimePopup.PopUpWindow (System.Int32 id) [0x0018a] in <4758f016d669411aa83fa2b6c8cb92c4>:0 
  at UnityEngine.GUILayout+LayoutedWindow.DoWindow (System.Int32 windowID) [0x00077] in <fa6f9762ac624af092525d37c9d516c4>:0 
  at UnityEngine.GUI.CallWindowDelegate (UnityEngine.GUI+WindowFunction func, System.Int32 id, System.Int32 instanceID, UnityEngine.GUISkin _skin, System.Int32 forceRect, System.Single width, System.Single height, UnityEngine.GUIStyle style) [0x00077] in <fa6f9762ac624af092525d37c9d516c4>:0 
UnityEngine.DebugLogHandler:Internal_LogException(Exception, Object)
UnityEngine.DebugLogHandler:LogException(Exception, Object)
ModuleManager.UnityLogHandle.InterceptLogHandler:LogException(Exception, Object)
UnityEngine.Logger:LogException(Exception, Object)
UnityEngine.Debug:CallOverridenDebugHandler(Exception, Object)
```

The above appears as soon as I click "Accept" for the first time in the window appears, but not for subsequent attempts until the window is closed/reopened. It can be reproduced repeatedly by clicking "Cancel" then right clicking the toolbar icon to re-open, then clicking "Accept," then cancel, re-open, accept, etc.

For kicks, I also tried creating an empty PopUpShown.cfg in the location that it seems to be pointing to, and it did not change the behavior.

**Comment by w3bward** [comment]
I fixed the issue I was having.

@SteveBenz , I was using a desktop shortcut with the popupwindow prompt. I assume whatever debug mode you're referring to is also a startup flag? 

I have multiple versions of KSP installed at the root of my C drive. To create the shortcut for my current game, I copied the shortcut I had for my 1.8.1 install, and updated the target, but I did not update "Start in" so I suspect that it was trying to use the relative path based off the directory defined in "Start in." Changing "Start in" to point at the correct KSP folder fixed the issue for me.

Assuming you're also using a shortcut to launch the game, make sure "Start in" is pointed at your KSP folder, and see if that resolves the problem?

![start_in](https://user-images.githubusercontent.com/2651605/87879086-4c936d00-c99d-11ea-99e0-bbb279c6e2ac.PNG)
